requirements.txt

```diff
@@ -43,7 +43,7 @@ progressbar==2.3
 # (http://docs.python-requests.org/en/latest/community/faq/#what-are-hostname-doesn-t-match-errors)
 ndg-httpsclient==0.3.0
 # Python markdown extensions for comment emails
-git+git://github.com/CenterForOpenScience/mdx_del_ins.git
+git+git://github.com/aleray/mdx_del_ins.git@242ec8077def0146a9e91f74d943cf49fe11ce43
```

Comment:
Drive-by: We don't want to depend on other people's GitHub repos, since we have no control over those (the user may force-push or even delete the repo).